webcomponents.js/webcomponents.js-tests.ts

```diff
@@ -10,10 +10,12 @@ var fooProto = Object.create(HTMLElement.prototype, {
     }
 });
 
-document.registerElement("x-foo", {
+var XFoo = document.registerElement("x-foo", {
     prototype: fooProto
 });
 
+var xFoo = new XFoo();
+
```

Comment:
you probably want to add some tests asserting that `HTMLElement` methods can be called on `xFoo`